### ISSUE TITLE
image with empty alt and global aria-attribute should not pass the image-alt rule

### ISSUE THREAD

**Issue by straker** [dequelabs/axe-core#2502]
The following code snippet should not pass the `image-alt` rule. The problem is that our `has-alt` check passes as the image has an empty alt, but the image is not considered presentational because of role conflict resolution of the global `aria-label` attribute.

```html
<img src="img.jpg" id="violation11" alt="" aria-label="" />
```

**Comment by straker** [comment]
Tested this in:

* Safari/VO
* IE11/JAWS
* Chrome/JAWS
* Firefox/NVDA

and all report the image as "blank" but not as an image element (just as in a newline character or empty tag). Thus this isn't an accessibility problem.
